realm/src/main/java/io/realm/Realm.java

```diff
@@ -895,6 +904,80 @@ public void writeCopyTo(File destination) throws IOException {
         return result;
     }
 
+    /**
+     * Converts a RealmObject that has been created using it's standard constructors instead of
+     * {@link #createObject(Class)} to a RealmObject that is managed by the Realm.
+     *
+     * To persist any further changes, the returned object must be be used.
```

Comment:
It might be worth adding a little documentation on what happens you continue to use the standalone object: changes to the persistent object in another thread will never be available to the standalone object - not even with auto-refresh.

Moreover, it should be emphasised that changes to a standalone object will not be persistent.

Finally, calling `copyToRealm` on the same standalone object will create two objects in the realm.